validator/schema_test.yml

```diff
@@ -80,6 +80,15 @@ object types:
       message: 'Name "__id" must not begin with "__", which is reserved by GraphQL introspection.'
       locations: [{line: 2, column: 3}]
 
+  - name: field argument list must not be empty
```

Comment:
This test merely documents the existing behavior, which is already correct (spec compliant). It's just for additional test coverage. The other two tests are for the fixes in this PR.